packages/firestore/src/exp/bundle.ts

```diff
@@ -46,6 +62,17 @@ export class LoadBundleTask
     documentsLoaded: 0
   };
 
+  /**
+   * Registers functions to listen to bundle loading progress events.
+   * @param next
```

Comment:
There should be a ` - ` after `@param next`. It is the format required by tsdoc.

Reply:
Done.